### ISSUE TITLE
Memory leak while playing back some gifs

### ISSUE THREAD

**Issue by szopin** [accumulator/Quickddit#28]
This one specifically:
https://www.reddit.com/r/CrappyDesign/comments/5szxfy/its_better_because_it_is_all_automated/
It's linked as jpg, while playing memory quickly reaches 97% usage and all apps get killed (jolla c, 2.1.0.9)

**Comment by accumulator** [comment]
This is a bug in QtQuick's AnimatedImage component. Quickddit merely passes this component the URL, so there's not much I can do. I've seen this in earlier sailfish versions as well..

**Comment by szopin** [comment]
Thanks, never seen this behaviour before (quickddit would hang on some gifs, but never crashed all bg apps), so thought it warranted a bug report. Thanks!

**Comment by Thaodan** [comment]
@accumulator do you know the qt bug too this?

**Comment by accumulator** [comment]
No, but this one comes closest: https://bugreports.qt.io/browse/QTBUG-55808

**Comment by Arvedui** [comment]
Maybe you can implement a workaround for imgur and others where a mp4 version is always available?

**Comment by accumulator** [comment]
relevant TJC : https://together.jolla.com/question/156463/memory-leak-when-viewing-gifs/

**Comment by accumulator** [comment]
I haven't seen any crashes the last few months. Also the linked image in szopin's original report is not crashing the app, therefore I'm closing this issue. Please add the reddit link to any problematic images if the problem still occurs!

**Comment by accumulator** [comment]
Hm, too soon, I hit this bug again :(
